src/app/pages/send/send-crypto-asset-form/components/amount-field.tsx

```diff
@@ -31,19 +31,21 @@ function getAmountModifiedFontSize(props: GetAmountModifiedFontSize) {
 }
 
 interface AmountFieldProps {
-  balance: Money;
-  switchableAmount?: JSX.Element;
-  bottomInputOverlay?: JSX.Element;
-  autofocus?: boolean;
   autoComplete?: 'on' | 'off';
+  autofocus?: boolean;
+  balance: Money;
+  bottomInputOverlay?: React.JSX.Element;
+  isSendingMax?: boolean;
+  switchableAmount?: React.JSX.Element;
   tokenSymbol?: string;
 }
 export function AmountField({
+  autoComplete = 'on',
```

Comment:
why is on default?

Reply:
@alter-eggo was this added by you for a specific reason? Why would we want this on?

Reply:
it was done a while ago... I think autocomplete was on in this component, and I just moved it to props with 'on' as default